### DISCUSSION TITLE
Welcome to indriya Discussions!

### DISCUSSION THREAD

**Discussion by keilw** [unitsofmeasurement/indriya#320] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tandcode** [comment]
Hi to everyone,
...
// _Moved to_ [_QA_](https://github.com/unitsofmeasurement/indriya/discussions/355)

**Reply by keilw** [reply]
Is there a chance you might place this in a slightly more prominent place like https://github.com/unitsofmeasurement/indriya/discussions/categories/q-a ?🤔
It may be a bit related to https://github.com/unitsofmeasurement/indriya/discussions/328 There is no direct way to relocate the discussien entry, so asking the question again and maybe deleting the comment here could help because this is just a "welcome" page explaining what the area is about.😇

**Reply by tandcode** [reply]
@keilw If it's needed I can remove my initial comment :sweat_smile:
